packages/web/src/hooks/useIsDevOrStaging.ts

```diff
@@ -0,0 +1,10 @@
+import { env } from 'services/env'
+
+export const useIsDevOrStaging = () => {
+  return (
+    env.ENVIRONMENT === 'development' ||
+    env.ENVIRONMENT === 'staging' ||
+    import.meta.env.DEV === true ||
```

Comment:
what's this import.meta.env hting

Reply:
vite way of detecting env

Reply:
nit: I would vote for `useEnvironment` or `useIsProduction` instead?